.env.devnet

```diff
@@ -61,6 +61,9 @@ L1_RPC_URL=http://localhost:8545
 # L2 RPC URL, e.g. Infura, Alchemy, or your own node
 L2_RPC_URL=http://localhost:8546
 
+# Provider url for testing web3 function with
+PROVIDER_URLS=http://localhost:8546
+
```

Comment:
Will this URL be different than what is `L2_RPC_URL`? Can we just use `L2_RPC_URL` without having a new one?

Reply:
In general it would be same. I am not sure where or which library picks this env var and we might have to change the lib code so maybe better to keep it separate (for now)